requirements.txt

```diff
@@ -1,2 +1 @@
-ops
```

Comment:
No ops?

Reply:
It comes included with charmcraft now (and charmcraft complains if we have it in the requirements.txt)

Reply:
Nice!